catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -817,4 +778,41 @@ protected void validateGetResourceRequest(ResourceRequest getResourceRequest)
           "Cannot perform getResource with null attribute value or null attributeName, either passed in from endpoint, or as output from PreResourcePlugin");
     }
   }
+
+  private URI getDerivedUri(Metacard metacard, String qualifier) throws ResourceNotFoundException {
+    List<URI> urisWithMatchingQualifier = new ArrayList<>();
+    Attribute derivedResourcesAttribute = metacard.getAttribute(Core.DERIVED_RESOURCE_URI);
+
+    if (derivedResourcesAttribute != null) {
+      for (Serializable ser : derivedResourcesAttribute.getValues()) {
+        if (ser instanceof String) {
+          try {
+            URI uri = new URI((String) ser);
+
+            if (uri.getFragment().equals(qualifier)) {
+              urisWithMatchingQualifier.add(uri);
+            }
+          } catch (URISyntaxException e) {
+            LOGGER.error(
```

Comment:
This should probably be at the debug level: https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels